src/main/java/org/betonquest/betonquest/compatibility/fakeblock/FakeBlockApiIntegrator.java

```diff
@@ -0,0 +1,39 @@
+package org.betonquest.betonquest.compatibility.fakeblock;
+
+import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.compatibility.Integrator;
+import org.betonquest.betonquest.exceptions.HookException;
+import org.bukkit.Server;
+
+/**
+ * Integrates with FakeBlock
+ */
+public class FakeBlockApiIntegrator implements Integrator {
+    /**
+     * The instance of {@link BetonQuest}.
+     */
+    private final BetonQuest plugin;
+
+    /**
+     * Create the FakeBlock integration.
+     */
+    public FakeBlockApiIntegrator() {
+        this.plugin = BetonQuest.getInstance();
+    }
+
+    @Override
+    public void hook() throws HookException {
+        final Server server = plugin.getServer();
```

Comment:
Please add a version check here. This makes sure that no one can load the integration with an older version of fakeblock that might not have the required API. See the `HologramIntegrator` for a reference implementation. 
https://github.com/BetonQuest/BetonQuest/blob/f4a136f2caaf84eb296ade9182680150e7f84151/src/main/java/org/betonquest/betonquest/compatibility/holograms/HologramIntegrator.java#L87-L90

Then the required version the the integration documentation.